go/systests/team_invite_test.go

```diff
@@ -199,3 +202,59 @@ func TestTeamInviteResetNoKeys(t *testing.T) {
 	// user 0 should get gregor notification that the team changed
 	tt.users[0].waitForTeamChangedGregor(team, keybase1.Seqno(3))
 }
+
+func prooveRooter(user *smuUser) {
+	tctx := user.primaryDevice().popClone()
```

Comment:
I think these clones are the issue.  You're probably getting a different G than the one for signup.  But I don't quite understand the clones...

Reply:
I tried disabling the clones as well (made popClone return clones[0] not remove it off the list) and it didn't help. but I haven't tried using `getPrimaryGlobalContext`. I'll try to write more tests, since I finally got this simple one to pass, and then try to experiment with the issue again.